dbt/adapters/athena/impl.py

```diff
@@ -133,6 +133,8 @@ class AthenaAdapter(SQLAdapter):
 
     quote_character: str = '"'  # Presto quote character
 
+    work_group_output_location_enforced: bool = None  # AWS WorkGroup cache
```

Comment:
let's also add a comment that we're not invalidating this cache, since workgroup settings updated rarely and it's fine to have these settings cached for dbt execution